en/docs/learn/configuring-the-bpm-profile-of-wso2-ei-as-a-workflow-engine.md

```diff
@@ -0,0 +1,92 @@
+
+    ```sql
+    <IS_HOME>/dbscripts/mysql.sql
+    <IS_HOME>/dbscripts/identity/mysql.sql
+    ```
+2.  Configure the user management database in both the IS and the BPM server. Add the following configurations with appropriate values in both the servers. Make sure that the URLs mention the name of the database you created in the previous step and the username and password are corresponding to your MySQL instance.     
+
+    1.  <IS_HOME>/repository/conf/deployment.toml 
+
+        ```toml
+        [database.identity_db]
+        type = "mysql"
+        url = "jdbc:mysql://localhost:3306/newdb?useSSL=false"
+        username = "root"
+        password = "Jan@1234"
+        driver = "com.mysql.jdbc.Driver"
+        validationQuery = "SELECT 1"
+        
+        [database.shared_db]
+        type = "mysql"
+        url = "jdbc:mysql://localhost:3306/newdb?useSSL=false"
+        username = "root"
+        password = "Jan@1234"
+        driver = "com.mysql.jdbc.Driver"
+        validationQuery = "SELECT 1"
+
+    
+    2.  <EI_HOME>/wso2/business-process/conf/datasources/master-datasources.xml
+
+        ```xml
+        <datasource>
+        <name>WSO2_CARBON_DB</name>
+        <description>The datasource used for registry and user manager</description>
+        <jndiConfig>
+            <name>jdbc/WSO2CarbonDB</name>
+        </jndiConfig>
+        <definition type="RDBMS">
+            <configuration>
+                <url>jdbc:mysql://localhost:3306/newdb</url>
+                <username>root</username>
+                <password>Jan@1234</password>
+                <driverClassName>com.mysql.jdbc.Driver</driverClassName>
+                <maxActive>80</maxActive>
+                <maxWait>60000</maxWait>
+                <minIdle>5</minIdle>
+                <testOnBorrow>true</testOnBorrow>
+                <validationQuery>SELECT 1</validationQuery>
+                <validationInterval>30000</validationInterval>
+            </configuration>
+        </definition>
+        </datasource> 
+        ```
+4.  Download the [MySQL JDBC driver](https://dev.mysql.com/downloads/connector/j/5.1.html) for the version you are using and       copy it to both `<IS_HOME>/repository/components/lib` and `<EI_HOME>/bin` folders.
+
+5.  Change the following configuration in the deployment.toml file in the `<IS_HOME>/repository/conf` directory. 
+    
+    ```toml
+    [user_store]
+    type = "database"
+    ```
+
+6.  Start WSO2 IS if you have not started it already and start the WSO2 EI business-process profile.
+
+    ```curl
+    cd <IS_HOME>/bin 
+    ./wso2server.sh
+    ```
+    
+    ```curl 
+    cd <EI_HOME>/bin
+    ./business-process.sh
+    ```
+7.  To test if the BPM profile has been configured properly to the Identity Server, create a user in the WSO2 Identity Server Management Console. Now when you navigate to the list of users in Business Process Server Management Console, the user created previously should be present. 
```

Comment:
To test if the BPM profile is properly configured in WSO2 Identity Server,

Reply:
Better to break this to multiple sub-points. 

Guidance: https://docs.google.com/document/d/14dZtE-f0BF46t5woX9k5aOawN1aIJ4WKaIFRqe7VyP8/edit#heading=h.i7s3q1sgw4a